services/catalog/src/simcore_service_catalog/db/events.py

```diff
@@ -27,6 +27,7 @@ async def connect_to_db(app: FastAPI) -> None:
             "server_settings": {"application_name": cfg.POSTGRES_CLIENT_NAME}
         },
         pool_pre_ping=True,  # https://docs.sqlalchemy.org/en/14/core/pooling.html#dealing-with-disconnects
+        future=True,
```

Comment:
MINOR: "Use the 2.0 style [Engine](https://docs.sqlalchemy.org/en/14/core/future.html#sqlalchemy.future.Engine) and [Connection](https://docs.sqlalchemy.org/en/14/core/future.html#sqlalchemy.future.Connection) API."

Q: when moving to 2, this is removed, right?

Reply:
yes